src/components/shared/Slider.tsx

```diff
@@ -76,32 +89,33 @@ export const Slider = ({
       >
         {data.results
           .filter(
-            (anime) => anime.status !== 'NOT_YET_RELEASED' && anime.coverImage
+            (anime) => anime.status !== "NOT_YET_RELEASED" && anime.coverImage
           )
           .map((anime) => (
-            <div key={anime.id}>
-              <div className="flex-none relative overflow-hidden">
-                <Link href={`/info/${anime.id}`}>
-                  <div className="transition-transform duration-300 ease-in-out hover:scale-[1.1] rounded-xl">
+              <div
+                key={anime.id}
+                className="min-w-[200px] max-h-full transition-all duration-500 ease-in-out hover:scale-105 rounded-xl shadow-lg "
+              >
+                <div className="flex-none relative overflow-hidden rounded-t-xl">
+                  <Link href={`/info/${anime.id}`}>
                     <Image
                       src={anime.coverImage!}
                       alt={anime.title.english ?? anime.title.romaji}
                       height={300}
                       width={200}
                       className="object-cover min-h-[285px] max-h-[285px] min-w-[200px]"
                     />
-                  </div>
-                </Link>
-              </div>
-              <div>
-                <h1 className="truncate max-w-[200px]">
-                  {anime.title.english ?? anime.title.romaji}
-                </h1>
-                <p className="font-bold text-xs">
-                  {anime.format} | {anime.status} | {anime.season}
-                </p>
+                  </Link>
+                </div>
+                <div className="p-4">
+                  <h1 className="text-white truncate text-lg font-semibold max-w-[200px]">
```

Comment:
Remove `text-white` as shadcn will automatically change text color based on the theme